core/src/modules/ops/runs.ts

```diff
@@ -119,9 +119,9 @@ export namespace RunOps {
               },
             },
           }),
-          exported: await run.$count("imports", {
+          processed: await run.$count("imports", {
```

Comment:
This method is used on the run details page to show this chart
<img width="1478" alt="Screen Shot 2022-01-10 at 8 50 26 AM" src="https://user-images.githubusercontent.com/303226/148805170-8a98c43d-d812-4445-91b7-efd03bac1521.png">

Are there any changes needed in the UI, or do we just display whatever the API returns?

Reply:
![image](https://user-images.githubusercontent.com/4368928/148819509-a37672ad-899c-4d33-9025-8162f7286b21.png)

Looks ok even on old runs